### DISCUSSION TITLE
Implicit feedback for recommendation systems and combining with content filtering

### DISCUSSION THREAD

**Discussion by dminovski0** [online-ml/river#620] [Q&A] [answered]
In the examples, some methods use Matrix Factorization for Recommender Systems with the datasets.MovieLens100K(), which has explicit movie ratings. I need to make recommendations based on implicit feedback, like views, page visits. This works with ALS and another package, but that package doesn't have online (streaming) learning. Does River support work with implicit feedback?

Also, the ALS algorithm works mostly with user-item matrices, with no room for feature columns. Can River add content filtering to the collaborative filtering methods? There are examples with the Factorization Machines (FM) algorithm.

**Comment by MaxHalford** [comment, marked as answer]
This isn't part of River yet, sorry. You're welcome to dig into this topic and contribute though!
